eZ/Publish/API/Repository/SearchService.php

```diff
@@ -18,6 +18,23 @@
 interface SearchService
 {
     /**
+     * Capability constants for use with {@see ::supports()}.
+     *
+     * Will add constants for highlighting, spellchecking & suggest once there is at least one implementation proving api design.
+     *
+     * Custom feature flags for features not made official yet is possible by prefixing values of the flags with
+     * identifier for engine, and placing the constant on said search engine. However this kind of tight coupling
+     * between api consumer and specific search engines is not officially supported by the api (as it's then not the api)-
+     */
+    const CAPABILITY_SCORING = 1;
+    const CAPABILITY_FACETS = 2;
+    const CAPABILITY_CUSTOM_FIELDS = 3;
```

Comment:
General remark about all the constants: I'd rather use power of 2, so it can be stored into single `int` and simplify `supports` implementation.